django/db/models/query.py

```diff
@@ -1263,12 +1262,10 @@ def columns(self):
         columns = self.query.get_columns()
         # Adjust any column names which don't match field names
         for (query_name, model_name) in self.translations.items():
-            try:
+            with suppress(ValueError):
+                # Ignore translations for nonexistent column names
```

Comment:
It feels more natural to me to put these type of comments before with statement rather than after it. Do you have any feelings about that?